examples/03_renderable/main.rs

```diff
@@ -6,13 +6,14 @@ extern crate cgmath;
 
 use amethyst::{Application, ElementState, Event, State, Trans, VirtualKeyCode, WindowEvent};
 use amethyst::asset_manager::{AssetManager, DirectoryStore};
-use amethyst::config::Element;
```

Comment:
I like this change, `project::Config` is somewhat more self-descriptive than `config::Element`.